.ci/packer_cache.sh

```diff
@@ -0,0 +1,14 @@
+# shellcheck disable=SC1091
+source /usr/local/bin/bash_standard_lib.sh
+
+readonly NODEJS_VERSION="12"
+
+DOCKER_IMAGES="docker.elastic.co/observability-ci/node-playwright:${NODEJS_VERSION}
+docker.elastic.co/observability-ci/node-puppeteer:${NODEJS_VERSION}"
+
+if [ -x "$(command -v docker)" ]; then
+  for di in ${DOCKER_IMAGES}
+  do
+  (retry 2 docker pull "${di}") || echo "Error pulling ${di} Docker image, we continue"
```

Comment:
is docker build also required to cache the docker images?

Reply:
I haven't seen it in other places: https://github.com/elastic/apm-pipeline-library/blob/master/.ci/packer_cache.sh